code/game/objects/items/melee/misc.dm

```diff
@@ -329,6 +337,185 @@
+				log_combat(user, target, "stunned", src)
+				target.visible_message(desc["visibleleg"], desc["localleg"])
+				if (Ll.get_staminaloss() < 26 && Ra.get_staminaloss() < 26 && La.get_staminaloss() < 26)
+					target.apply_damage(25, STAMINA, BODY_ZONE_R_LEG, def_check)
+				else
+					target.apply_damage(10, STAMINA, BODY_ZONE_R_LEG, def_check)
+				if (Rl.get_staminaloss() == 50)
+					target.apply_damage(10, STAMINA, BODY_ZONE_CHEST, def_check)
+				else
+					target.apply_damage(5, STAMINA, BODY_ZONE_CHEST, def_check)
+
+				if(Rl.get_staminaloss() == 50 && CHECK_BITFIELD(target.mobility_flags, MOBILITY_STAND) && (!trip_cooldowns[target] || COOLDOWN_FINISHED(src, trip_cooldowns[target])))
+					target.visible_message("<span class='emote'><b>[T]</b> [pick(list("falls down.","falls face first into the floor.","gets viciously tripped.","got clumsy."))]</span>")
+					target.balloon_alert_to_viewers("Tripped!")
+					target.Knockdown(7)
+					log_combat(user, target, "tripped", src)
+					playsound(usr.loc, "sound/misc/slip.ogg", 30, 1)
+					target.visible_message(desc["visibletrip"], desc["localtrip"])
+					COOLDOWN_START(src, trip_cooldowns[target], 3 SECONDS)
+
+			else if(user.is_zone_selected(BODY_ZONE_L_ARM))
+				if(!La.get_staminaloss() == 50)
+					log_combat(user, target, "stunned", src)
+					target.visible_message(desc["visiblearm"], desc["localarm"])
+				else
+					log_combat(user, target, "disarmed", src)
+					target.visible_message(desc["visibledisarm"], desc["localdisarm"])
+				if (Ra.get_staminaloss() < 26 && Ll.get_staminaloss() < 26 && Rl.get_staminaloss() < 26)
+					target.apply_damage(20, STAMINA, BODY_ZONE_L_ARM, def_check)
+				else
+					target.apply_damage(5, STAMINA, BODY_ZONE_L_ARM, def_check)
+				if (La.get_staminaloss() == 50)
+					target.apply_damage(10, STAMINA, BODY_ZONE_CHEST, def_check)
+				else
+					target.apply_damage(4, STAMINA, BODY_ZONE_CHEST, def_check)
+
+			else if(user.is_zone_selected(BODY_ZONE_R_ARM))
+				if(!Ra.get_staminaloss() == 50)
+					log_combat(user, target, "stunned", src)
+					target.visible_message(desc["visiblearm"], desc["localarm"])
+				else
+					log_combat(user, target, "disarmed", src)
+					target.visible_message(desc["visibledisarm"], desc["localdisarm"])
+				if (La.get_staminaloss() < 26 && Ll.get_staminaloss() < 26 && Rl.get_staminaloss() < 26)
+					target.apply_damage(20, STAMINA, BODY_ZONE_R_ARM, def_check)
+				else
+					target.apply_damage(5, STAMINA, BODY_ZONE_R_ARM, def_check)
+				if (Ra.get_staminaloss() == 50)
+					target.apply_damage(10, STAMINA, BODY_ZONE_CHEST, def_check)
+				else
+					target.apply_damage(4, STAMINA, BODY_ZONE_CHEST, def_check)
+
+			add_fingerprint(user)
+
+			if(!iscarbon(user))
+				target.LAssailant = null
+			else
+				target.LAssailant = WEAKREF(user)
+			COOLDOWN_START(src, cooldown_check, cooldown)
+		else
+			var/wait_desc = get_wait_description()
+			if (wait_desc)
+				to_chat(user, wait_desc)
```

Comment:
Instead of having these else block at the end here, you can simplify and make the control flow more readable:

```
else if(cooldown_check > world.time)
    var/wait_desc = get_wait_description()
    if (wait_desc)
        to_chat(user, wait_desc)
    return
if(ishuman(target))
     //....
     if(H.check_shields(src, 0, "[user]'s [name]", MELEE_ATTACK))
          return
// check bodyzones ...
```

Reply:
I'll adress your concerns when im back to this project again!